### DISCUSSION TITLE
Can I use more than one model and tables

### DISCUSSION THREAD

**Discussion by lukeboy2002** [filamentphp/filament#8161] [Help]
I want to add manual members to my site.
I have a model users and a model members.
In my model users I add the username, email etc. When I create a member it get the role of member. 
My input fields are:
Username:
email:
password and conform password
image.

In filament i want to add de username, mail and password to the database user and create a member with user_id and the images.

Is this possible and how.

**Comment by leandrocfe** [comment]
[Filament v3 - customizing-the-creation-process](https://filamentphp.com/docs/3.x/panels/resources/creating-records#customizing-the-creation-process)
